test/internal/laws/chain_.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var R = require('ramda');
+
+//  flatMap :: Chain m => (a -> m b) -> m a -> m b
+module.exports = function flatMap(f) {
```

Comment:
Good catch. To avoid a naming conflict on case-insensitive file systems I originally named this file __flatMap.js__ (__chain.js__ is not an option due to the presence of __Chain.js__). I later decided to go with __chain\_.js__ instead, but forgot to update these.

:zap: